scan.py

```diff
@@ -62,7 +62,7 @@ def set_av_tags(s3_object, result):
     curr_tags = s3_client.get_object_tagging(Bucket=s3_object.bucket_name, Key=s3_object.key)["TagSet"]
     new_tags = copy.copy(curr_tags)
     for tag in curr_tags:
-        if tag["Key"] == AV_STATUS_METADATA:
+        if tag["Key"] in [AV_STATUS_METADATA, AV_TIMESTAMP_METADATA]:
             new_tags.remove(tag)
             break
```

Comment:
Now that there's two I think the `break` needs removed... might be faster to actually just loop over the tags we want to remove and remove them if they exist. NBD on that though.